src/gui/Src/Gui/LogView.cpp

```diff
@@ -375,30 +379,56 @@ void LogView::clearLogSlot()
     this->clear();
 }
 
+void LogView::redirectLogSlotStop()

```

Comment:
The `Slot` should be put at the end of the function name to avoid confusion (same for the other functions).

Reply:
I renamed to `stopRedirectLogSlot`.